Assets/Settings/SampleScenePostProcessingSettings.asset

```diff
@@ -168,8 +168,8 @@ MonoBehaviour:
     m_OverrideState: 1
     m_Value: 10
   compensation:
```

Comment:
There's a known problem with compensation interfaces.
Support for Metal is horrible, but the game is mainly for PCs, so it's not that big of a deal.

Performance key for Metal: `EXPERIMENTAL_METAL_PROGRESSIVE_LIGHT_COMPRESSION`
Performance API call: `exp::partial_support_get__apis()-->metal_light_compression`